twython/endpoints.py

```diff
@@ -199,7 +199,28 @@ def upload_video(self, media, media_type, size=None):
             'command': 'FINALIZE',
             'media_id': media_id
         }
-        return self.post(upload_url, params=params)
```

Comment:
Instead of changing the return here, can you add a keyword argument  to `upload_video` like `check_progress=False` and if they pass `check_progress=True`, it will execute the code you added, otherwise it will just return the response.